package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "execa": "^6.1.0",
     "meow": "^10.1.2",
-    "octokit": "^1.7.2"
+    "octokit": "^3.1.2"
```

Comment:
`npm audit` reports 2 high vulnerability, 1 of which is for `octokit < 3.1.2` and `3.1.2` has support for node 18 and 20 only.

<img width="1322" alt="Screenshot 2024-01-22 at 11 26 37 AM" src="https://github.com/10up/changelog-generator/assets/17757960/9f40a7d5-5598-461e-a311-e39e63147cbf">

I just checked that Gutenberg's development environment [is set to node v20](https://github.com/WordPress/gutenberg/blob/trunk/.nvmrc). Should we start discussing around the topic of updating node version across our repo to v18 at least?

cc: @10up/open-source-practice